### ISSUE TITLE
Crash During Client Loading

### ISSUE THREAD

**Issue by xlxAciDxlx** [Team-RTG/Realistic-Terrain-Generation#1086]
##### Affected Mod Versions:
- AbyssalCraft: 1.9.3.9
- Biomes O Plenty: 5.0.0.2109
- Forge version: 12.18.3.2185
- RTG version: 4.1.2.0

##### Issue Description:
Disabling an AbyssalCraft biome in the config causes RTG to crash.

##### Steps to reproduce the issue: (If it's easily reproducible.)
1. Install AbyssalCraft
2. Install Biomes O Plenty
3. Install RTG
4. Disable an AbyssalCraft biome in the config
5. Launch & crash, no profit :(

##### Extra Information:
- [Crash Log](https://gist.github.com/xlxAciDxlx/1c19407478fcd7296eb8d5adfb483e06)
- [FML Client Latest.log](https://gist.githubusercontent.com/xlxAciDxlx/5410a961eef492e8a48b6c7fbf8029ab/raw/8c2e4b12f6987ce7f33f16d55f89b8fecc7737f6/fml-client-latest.log)

**Comment by srs-bsns** [comment]
Please provide the full crash report, and your `fml-client-latest.log`

**Comment by xlxAciDxlx** [comment]
Updated the initial post to include both. Sorry about that!

**Comment by xlxAciDxlx** [comment]
So, I'm pretty sure this has something to do with RTG trying to add the "Darklands Highland" from AbyssalCraft, as that shows up in the client log and that biome is actually disabled in my AbyssalCraft config. Will try enabling it in the config and see if it fixes the crash.

Yep, this is definitely confirmed to be caused by AbyssalCraft biomes being disabled via the config. I only re-enabled one biome and left the others disabled, and the next client log was changed to one of the two that was disabled.

---
I also updated my main post again so others can produce this right off the bat, rather than having to read through the other comments. Hope that helps!
